sentry-spring/src/main/java/io/sentry/spring/SentryInitBeanPostProcessor.java

```diff
@@ -68,4 +73,9 @@ public void setApplicationContext(final @NotNull ApplicationContext applicationC
       throws BeansException {
     this.applicationContext = applicationContext;
   }
+
+  @Override
+  public void destroy() {
+    Sentry.close();
```

Comment:
Should we have a ctor that takes an `IHub` and use the default ctor with the `HubAdapter` approach? so we can test the `close` call.